content/docs/services/cloud-gov-identity-provider.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: cloud.gov identity provider
 name: "cloud-gov-identity-provider"
-description: "Manage client credentials for authenticating cloud.gov users in your app"
+description: "Authenticate cloud.gov users in your app"
```

Comment:
We already had a meeting specifically to nail down this text. I'm wondering how much more time this is worth--are we making this clearer or talking in circles?